### PR TITLE
Disable docker under act

### PR DESCRIPTION
**Pull request by grahamc**
##### Checklist

- [ ] Tested changes against a test repository
- [ ] Added or updated relevant documentation (leave unchecked if not applicable)
- [ ] (If this PR is for a release) Updated README to point to the new tag (leave unchecked if not applicable)

### CODE REVIEW

**Review by wh0 — APPROVED** [review]
docker shim doesn't work in act due to inability to mount /nix from a container rather from act's host https://github.com/nektos/act/issues/410

File: src/index.ts [hunk 1]
```diff
@@ -428,7 +435,7 @@ class NixInstallerAction {
       extraConf += `access-tokens = ${serverUrl}=${this.githubToken}`;
       extraConf += "\n";
     }
-    if (this.trustRunnerUser !== null) {
+    if (this.trustRunnerUser !== null && this.trustRunnerUser) {
```
Comment (lucperkins) [inline src/index.ts:438]: `getBool` returns a Boolean, so you can update the type of `trustRunnerUser` to `boolean` rather that `boolean | null` and thus you don't need the null check here.

Comment (lucperkins) [inline src/index.ts:438]: getBool returns a Boolean, so you can update the type of trustRunnerUser to boolean rather that boolean | null and thus you don't need the null check here.

**Review by lucperkins — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (grahamc) [reply src/index.ts]: Nice, fixed!

**Review by lucperkins — APPROVED** [review]

**Pull request merged**